masjid_util.py

```diff
@@ -0,0 +1,46 @@
+_MASJID_METADATA = {
+  'MCA': {
```

Comment:
let's follow the example of util.py and have an enum for all of our masajid and then key this map based on the enum